src/service/resources-impl.js

```diff
@@ -1495,8 +1476,7 @@ export class Resource {
     /** @const {!AmpElement|undefined|null} */
     this.owner_ = undefined;
 
-    /** @const {number} */
-    this.priority_ = getElementPriority(element.tagName);
+    this.priority_ = element.getElementPriority();
```

Comment:
Currently, it's:
```
Resource.prototype.getPriority = function() {
  return this.priority_;
}
```

The best I can gather right now, it will instead become:
```
Resource.prototype.getPriority = function() {
  return this.element.getPriority();
}

ElementProto.getPriority = function() {
  dev.assert(this.isUpgraded(), 'Must be upgraded');
  return this.implementation_.getPriority();
}
```

And obviously, `Resource.priority_` interval variable will be gone completely.